ci/rh-olm-release-pipeline.yaml

```diff
@@ -0,0 +1,400 @@
+---
+
+resources:
+  - name: agent-operator-pipeline-source
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/instana/instana-agent-operator.git
+      branch: main
+
+  - name: instana-agent-operator-release
```

Comment:
I have seen that a resource rate limit happens for this and for the `yq-release`.

One possibility would be to use a token to get a higher limit. Or we can set `check_every` and change the check to every hour or a higher value. The default is one minute, and there are certainly many resources outside our team.
So it is not this pipeline, it's the total number of resources using the Github API
https://concourse-ci.org/resources.html#schema.resource.check_every